app/views/studies/_shared_sync_functions.js.erb

```diff
@@ -95,6 +167,31 @@ $('#study-file-<%= study_file.id %>').on('change', '.file-type', function() {
             submitBtn.attr('disabled', false);
         }
     }
+
+    // FileUploadControl updates this more elegantly via React,
+    // but this direct approach is easier and as effective here.
+    let useConvention = false
+    if (fileType === 'Metadata') {
+      const conventionSelector =
+        '#study-file-<%= study_file.id %> ' +
+        '#study_file_use_metadata_convention_true'
+      useConvention = ($(conventionSelector)[0].checked === true)
+    }
+    const fileOptions = {
+      use_metadata_convention: useConvention
+    }
+
+    const bucketName = '<%= @study.bucket_id %>'
+    const results = await window.SCP.validateRemoteFileContent(
+      bucketName, fileName, fileType, fileOptions
+    );
+
+    const issuesHtml = getIssuesHtml(results, fileName, fileId)
```

Comment:
as mentioned above, put an <div id="validation-results-{fileId}></div> on the form, and then just use a render helper method to put it on the form

Reply:
This makes sense to do as part of a separate task for `ValidationMessage`.